docs/docs/dev_docs/tutorials/token_portal/setup.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Setup and Installation
+---
+
+In this step, we’re going to
+
+1. Install prerequisites
+2. Create a yarn project to house everything
+3. Create a nargo project for our Aztec contract
+4. Create a hardhat project for our Ethereum contract(s)
+5. Import all the Ethereum contracts we need
+6. Create a yarn project for our TS
+
+We recommend going through this setup to fully understand where things live.
+
+However if you’d rather skip this part, our dev-rels repo contains the starter code here.
+
+# Prerequisites
+
+- [node v18+](https://github.com/tj/n)
+- [docker](https://docs.docker.com/)
+- [Aztec sandbox](https://docs.aztec.network/dev_docs/getting_started/sandbox)
+
+```bash
+/bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
+```
+
+- Nargo
+
+```bash
+curl -L https://raw.githubusercontent.com/noir-lang/noirup/main/install | bash
+noirup -v 0.12.0-aztec.0
+```
+
+# Create a yarn project
+
+Our root yarn project will house everything ✨
+
+```bash
+mkdir aztec-token-portal && cd aztec-token-portal && yarn init
```

Comment:
I prefer refactoring all of this to be called "bridge" rather than "portal"
Portal just refers to the L1 counterpart. Whereas bridge sounds more holistic.